xbmc/cores/AudioEngine/Sinks/AESinkALSA.cpp

```diff
@@ -1046,10 +1046,12 @@ bool CAESinkALSA::OpenPCMDevice(const std::string &name, const std::string &para
       case 4:
         if (TryDeviceWithParams("surround40" + openName, params, pcmp, lconf))
           return true;
+        [[fallthrough]]
```

Comment:
What actually will fall through really?

Reply:
no idea, i just got the warning... and forgot to add `;`

Reply:
If the switch is entered with 3 channels and surround40 fails it will fall through, which you need to explicitly tell with the [[fallthrough]]
So makes some sense to me.
Not sure what TryDeviceWithParams does, but even if it's hypothetical the fallthrough makes the intention obvious.